binding/Binding.Shared/PlatformConfiguration.cs

```diff
@@ -50,18 +50,20 @@ static PlatformConfiguration ()
 			Is64Bit = IntPtr.Size == 8;
 		}
 
+		public static bool? IsMuslOverride { get; set; }
```

Comment:
Technically there are other libc implementations, so I'd suggest to have something like 
```cs
enum LibcFlavor
{
    Glibc,
    Musl
}
```

Reply:
OK, so I changed the whole way of selecting a variant/flavor.

There is now a property `LinuxFlavor` that can be used to control the whole thing. This is useful in cases where there needs to be a specific version of Linux. For example if you need a custom build for say FreeBSD. You can set the flavor at app startup following any checks that you do and then the lookup will use that first.

If the property is null, then I just do the basic glibc/musl check - or rather I check for glibc and assume musl otherwise. If a specific flavor is set, then I do nothing and just use that. This is just a plain string that allows for extension later on, and also is just the name of the folder to use.

Example usage would be on a x64 machine:

- `LinuxFlavor == null` with glibc => 'x64'
- `LinuxFlavor == null` with non-glibc => 'musl-x64'
- `LinuxFlavor == freebsd` => 'freebsd-x64'

Since this is totally customizable, you can even do specific checks for versions and set the flavor to `opensuse.14` and then it will look in `opensuse.14-arm64` if it were running on an ARM64 machine.